server/service/src/service_provider.rs

```diff
@@ -1,68 +1,43 @@
         contact_trace::{ContactTraceService, ContactTraceServiceTrait},
         encounter::{EncounterService, EncounterServiceTrait},
         patient::{PatientService, PatientServiceTrait},
         program_enrolment::{ProgramEnrolmentService, ProgramEnrolmentServiceTrait},
         program_event::{ProgramEventService, ProgramEventServiceTrait},
-    },
-    repack::{RepackService, RepackServiceTrait},
-    report::report_service::{ReportService, ReportServiceTrait},
-    requisition::{RequisitionService, RequisitionServiceTrait},
-    requisition_line::{RequisitionLineService, RequisitionLineServiceTrait},
-    rnr_form::{RnRFormService, RnRFormServiceTrait},
-    sensor::{SensorService, SensorServiceTrait},
-    settings_service::{SettingsService, SettingsServiceTrait},
-    stock_line::{StockLineService, StockLineServiceTrait},
-    stocktake::{StocktakeService, StocktakeServiceTrait},
-    stocktake_line::{StocktakeLineService, StocktakeLineServiceTrait},
-    store::{get_store, get_stores},
-    sync::{
+    }, repack::{RepackService, RepackServiceTrait}, report::report_service::{ReportService, ReportServiceTrait}, requisition::{RequisitionService, RequisitionServiceTrait}, requisition_line::{RequisitionLineService, RequisitionLineServiceTrait}, rnr_form::{RnRFormService, RnRFormServiceTrait}, sensor::{SensorService, SensorServiceTrait}, settings_service::{SettingsService, SettingsServiceTrait}, stock_line::{StockLineService, StockLineServiceTrait}, stocktake::{StocktakeService, StocktakeServiceTrait}, stocktake_line::{StocktakeLineService, StocktakeLineServiceTrait}, store::{get_store, get_stores}, sync::{
         site_info::{SiteInfoService, SiteInfoTrait},
         sync_status::status::{SyncStatusService, SyncStatusTrait},
         synchroniser_driver::{SiteIsInitialisedTrigger, SyncTrigger},
-    },
-    temperature_excursion::{TemperatureExcursionService, TemperatureExcursionServiceTrait},
-    vaccine_course::VaccineCourseServiceTrait,
-    ListError, ListResult,
+    }, temperature_excursion::{TemperatureExcursionService, TemperatureExcursionServiceTrait}, vaccine_course::VaccineCourseServiceTrait, ListError, ListResult
 };
 use repository::{
     PaginationOption, RepositoryError, StorageConnection, StorageConnectionManager, Store,
     StoreFilter, StoreSort,
 };
+use rust_embed::RustEmbed;
+
+// Define a struct for development
+#[cfg(debug_assertions)]
+#[derive(RustEmbed)]
+#[folder = "../../client/packages/common/src/intl/locales"] // Path for development
+struct DevLocalisations;
+
+
+// Define a struct for production
+#[cfg(not(debug_assertions))]
+#[derive(RustEmbed)]
+#[folder = "../../client/packages/host/dist/locales"] // Path for production
```

Comment:
Pretty sure we can just have a single implementation here.
At build time the client packages should be available, and would simplify this...

Reply:
The JSON for translations doesn't get modified in the webpack build process as far as I know.